metadata/app/schema.go

```diff
@@ -29,8 +29,12 @@ const (
 	DefaultVersion    = "0.0.1"
 )
 
-var ErrRegistryNameInvalid = fmt.Errorf("Registry name is invalid")
-var ErrRegistryExists = fmt.Errorf("Registry with name already exists")
+var (
+	ErrRegistryNameInvalid    = fmt.Errorf("Registry name is invalid")
```

Comment:
golint will complain because these exported items don't have a comment.

Reply:
This is existing code from the Registry portions of the schema. I've added comments on all new exported Environment Spec related code. Since registries are being re-written I didn't want to spend time to fix existing lint errors related to registries, and wanted to keep this change contained to Environments / Components.